dashboard/db/migrate/20190321174350_backfill_school_year_in_ap_school_codes.rb

```diff
@@ -0,0 +1,5 @@
+class BackfillSchoolYearInApSchoolCodes < ActiveRecord::Migration[5.0]
+  def change
+    Census::ApSchoolCode.where(school_year: nil).update_all('school_year = 2016')
```

Comment:
is there a difference between doing this and doing `update_all(school_year: "2016")`?

Reply:
Agreed.  I default to using the ActiveRecord approach rather than having snippets of raw SQL text.  By the way ... `school_year` is an integer, so I think it would be:
`update_all(school_year: 2016)`

Reply:
Per docs I don't think so, I think it'd be `school_year: 2016` since it's an int? Could switch to that though if more ruby-ish?

Reply:
I have no preference either way, I'm just curious if there's a difference